retrieval/target.go

```diff
@@ -359,9 +364,15 @@ func (t *Target) scrape(sampleAppender storage.SampleAppender) (err error) {
 	}()
 
 	for samples := range t.ingestedSamples {
-		for _, s := range samples {
-			s.Metric.MergeFromLabelSet(baseLabels, clientmodel.ExporterLabelPrefix)
-			sampleAppender.Append(s)
+		if t.proxy {
+			for _, s := range samples {
+				sampleAppender.Append(s)
```

Comment:
@beorn7 For the "Federation: No label is auto-attached" case, we still want to at least apply labels that are set for the target in the config, right?